apps/src/templates/studioHomepages/CourseBlocksStudentGradeBands.jsx

```diff
@@ -14,16 +14,19 @@ class CourseBlocksStudentGradeBands extends Component {
     {
       heading: i18n.courseBlocksGradeBandsK5(),
       description: i18n.courseBlocksGradeBandsK5Description(),
+      buttonText: i18n.courseBlocksGradeBandsK5Button(),
       path: '/student/elementary'
     },
     {
       heading: i18n.courseBlocksGradeBands612(),
       description: i18n.courseBlocksGradeBands612Description(),
+      buttonText: i18n.courseBlocksGradeBands612Button(),
       path: '/student/middle-high'
     },
     {
       heading: i18n.courseBlocksGradeBandsUniversity(),
       description: i18n.courseBlocksGradeBandsUniversityDescription(),
+      buttonText: i18n.courseBlocksGradeBandsUniversity(),
```

Comment:
Is there a reason there isn't a separate Button for this one? I feel like `Beyond K-12` isn't descriptive enough. Thoughts about `Opportunities Beyond K-12`?

Reply:
I made one before but wasn't sure what to put there because 'courses' didn't feel right! I like this!

Reply:
Okay I updated the button and the picture in the PR description:)

Reply:
Looks great!